gammapy/irf/psf/parametric.py

```diff
@@ -263,7 +263,7 @@ def get_sigmas_and_norms(**kwargs):
     sigmas = u.Quantity([kwargs[f"sigma_{idx}"] for idx in [1, 2, 3]])
 
     scale = kwargs["scale"]
-    ones = np.ones_like(scale)
+    ones = np.ones_like(scale).value
```

Comment:
`ones = np.ones(scale.shape)` is a bit shorter, but OK to keep what you have.

Reply:
Ok.